Forge.TreeWalker.UnitTests/test/InvalidTestSchemas/InvalidTestSchemaErrorContent.json

```diff
@@ -0,0 +1,29 @@
+{
+    "Tree": {
+        "Root": {
+            "Type": "Action",
+            "Actions": {
+                "Root_CollectDiagnosticsAction": {
+                    "Action": "CollectDiagnosticsAction",
+                    "Input": {
+                        "Command": "TheCommand"
+                    }
+                }
+            },
+            "ChildSelector": [
+                {
+                    "Child": "LeafNodeSummaryTest"
+                }
+            ]
+        },
+        "LeafNodeSummaryTest": {
+            "Type": "Leaf",
+            "Actions": {
+                "LeafNodeSummaryTest_LeafNodeSummaryAction": {
+                    "Action": "LeafNodeSummaryAction",
+                    "Input": "C#|Session.GetLastActionResponse()"
+                }
+            }
+        }
+    }
+}
```

Comment:
Let's change this test up a bit. We actually have a backlog item to remove "Label" as a required property in ChildSelector. Let's go ahead and do that in this PR since we're already messing with the validations rules a bit. This is on line 178 of ForgeSchemaValidationRules.json.

We can test something else here. How about LeafNodeSummaryAction without an Input. That should be a stable test. Make the schema as minimal as possible. Adding extra fluff like an Action node just distracts from what you are trying to test.

Reply:
Please re-read this comment. Still a couple actions to take in iteration 6.

Reply:
This is what I meant. Please use this and add closing brackets.

{
    "Tree": {
        "Root": {
            "Type": "Action",
            "Actions": {
                "LeafActionWithoutRequiredActionInput": {
                    "Action": "LeafNodeSummaryAction"

Reply:
I made a little change on that: "Type": "Action" -> "Type": "Leaf", otherwise it would be regarded as a ActionTypeNode and pass the validation

Reply:
Correct, good catch.